dist/preview release/what's new.md

```diff
@@ -37,3 +37,19 @@
 
 
 ## Breaking changes
+
+My Pr simplifies code contributions by fully automating the dev setup with gitpod, a free online VS Code like IDE with a single click it'll launch a ready-to-code workspace where:
```

Comment:
I guess a one liner with your github alias outside the breaking change section would make more sense.